src/clr-angular/forms/datepicker/date-container.ts

```diff
@@ -131,7 +133,9 @@ export class ClrDateContainer implements DynamicWrapper, OnDestroy {
 
   @HostListener('body:keyup.escape')
   close(): void {
-    this.toggleButton.nativeElement.focus();
+    if (this.isOpen === true) {
```

Comment:
Could this be reduced to just:
```
if (this.isOpen) { ... } 
```

Reply:
Sure, 
I have this mental note in my head to write sometimes like that to simplify my head compiler :)

will push fix tomorrow morning